woocommerce/woocommerce-gateway-stripe
# feat: add Stripe setttings base layout

## Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1706

Adds some base layout to the settings page.
I included a few packages:
- Emotion (used also by WP components)
- WP components
- WP i18n

Most of the styles/names come from WCPay.

## Testing instructions

- Ensure you have the `_wcstripe_feature_upe_settings` option in `wp_options` set to `'1'`
- Ensure your dependencies are up to date
- Run `npm run start`
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- A new layout should appear

Before:
![Screen Shot 2021-08-19 at 12 09 40 PM](https://user-images.githubusercontent.com/273592/130112997-f108e227-97a3-4cfe-ae50-05abd223ea27.png)

After:
![Screen Shot 2021-08-20 at 8 49 33 AM](https://user-images.githubusercontent.com/273592/130243460-1939d71e-78ab-4fdc-aef1-0271bd563d38.png)
